include.js

```diff
@@ -5904,8 +5904,16 @@ let cards = [
     author: 'Viskarra',
     githubLink: 'https://github.com/viskarra'
   }
```

Comment:
Please fix an error by adding a comma before your object details @Mayorman07

Reply:
this has been fixed now, thank you for pointing it out @donmarvex

Reply:
@Mayorman07 could you create another pull request with all you work